tests/integration/it_locations_test.py

```diff
@@ -37,16 +37,33 @@ def test_create(client):
 
 
 def test_edit(client):
-    assert client.get("/locations/edit/1").status_code == HTTPStatus.OK
+    location_old = factories.LocationFactory()
+    location_data = {
+        "name": "Name",
+        "code": "Code",
+        "country": "Country",
+        "region": "Region",
+        "city": "City",
+        "address": "Address",
+        "longitude": "0.0000000",
+        "latitude": "0.0000000",
+        "type": "Type",
+        "status": "Active",
+        "comment": "No comments",
+    }
+    url = url_for("location.edit", id=location_old.id)
+    response = client.post(url, data=location_data)
+    assert response.status_code == HTTPStatus.FOUND
+    assert response.location.endswith(url_for('location.list'))
+    assert Location.query.count() == 1
+    location = Location.query.get(location_old.id)
+    for attr in location_data.keys():
+        assert getattr(location, attr) == location_data[attr]
 
 
 def test_delete(client):
-    response = client.post("/locations/delete", data={"id": 1})
-    assert response.headers["Location"] == "http://localhost/locations/"
-
-
-def _create_test_location():
-    return Location(
-        name="Test location", code="1", country="USA", region="West",
-        city="LA", type="Type", address="Address", longitude="11223344",
-        latitude="44332211", status="T")
+    location = factories.LocationFactory()
+    url = url_for("location.delete", id=location.id)
+    response = client.post(url)
+    assert response.status_code == HTTPStatus.FOUND
+    assert response.location.endswith(url_for('location.list'))
```

Comment:
@vladarefiev can you also check that the response is actually empty? only 1 location has been introduced and then removed, so the list should be empty.

Reply:
@amihaiemil agree with you, but I prefer to check that data was deleted from DB. WDYT?

Reply:
@vladarefiev ok :)